src/payment/checkout/Checkout.jsx

```diff
@@ -176,67 +213,34 @@ class Checkout extends React.Component {
             {/* Apple Pay temporarily disabled per REV-927  - https://github.com/openedx/frontend-app-payment/pull/256 */}
           </p>
         </div>
-        {loadingCyberSource || loadingStripe
-          ? (
-            <>
-              <div className="skeleton py-1 mb-3 w-25" />
-              <div className="row">
-                <div className="col-lg-6">
-                  <div className="skeleton py-3 mb-3" />
-                  <div className="skeleton py-3 mb-3" />
-                  <div className="skeleton py-3 mb-3" />
-                  <div className="skeleton py-3 mb-3" />
-                </div>
-                <div className="col-lg-6">
-                  <div className="skeleton py-3 mb-3" />
-                  <div className="skeleton py-3 mb-3" />
-                  <div className="skeleton py-3 mb-3" />
-                  <div className="skeleton py-3 mb-3" />
-                </div>
-              </div>
-              <div className="skeleton py-1 mb-3 mt-5 w-25" />
-              <div className="row">
-                <div className="col-lg-6">
-                  <div className="skeleton py-3 mb-3" />
-                  <div className="skeleton py-3 mb-3" />
-                </div>
-                <div className="col-lg-6">
-                  <div className="skeleton py-3 mb-3" />
-                  <div className="skeleton py-3 mb-3" />
-                </div>
-              </div>
-            </>
-          ) : null}
-        {!loading
-          && (enableStripePaymentProcessor
-            ? (options.clientSecret && (
-              <Elements options={options} stripe={stripePromise}>
-                <StripePaymentForm
-                  onSubmitPayment={this.handleSubmitStripe}
-                  onSubmitButtonClick={this.handleSubmitStripeButtonClick}
-                  clientSecret={options.clientSecret}
-                  disabled={submitting}
-                  isBulkOrder={isBulkOrder}
-                  isProcessing={stripeIsSubmitting}
-                  loading={loading}
-                  isQuantityUpdating={isQuantityUpdating}
-                />
-              </Elements>
-            )
-            )
-            : (
-              <PaymentForm
-                onSubmitPayment={this.handleSubmitCybersource}
-                onSubmitButtonClick={this.handleSubmitCybersourceButtonClick}
-                disabled={submitting}
-                loading={loading}
-                loaded={loaded}
-                isProcessing={cybersourceIsSubmitting}
-                isBulkOrder={isBulkOrder}
-                isQuantityUpdating={isQuantityUpdating}
-              />
-            )
-          )}
+
```

Comment:
Decided to separate rendering Stripe or CyberSource ternary previously done so that it's also easier to remove the CyberSource part in the future.

I am also not adding a billing form skeleton on the CyberSource payment flow since we don't have one right now, focusing on improving Zebra payment flow and not what's already there. This simplifies the conditionals I had before.

Reply:
Update: added `loading` so that it does show on the CyberSource payment flow. I'm curious to see how fast loading is for CyberSource that it might not even show the skeleton.